aggregation/simple/src/main/java/com/spotify/heroic/aggregation/simple/PointPairArea.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2015 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.aggregation.simple;
+
+import com.spotify.heroic.metric.Point;
+
+public class PointPairArea {
+    /**
+     * Compute the area that two data points form with the X-axis. The area is positive above
+     * the X-axis and negative below.
+     *
+     * @param a first data point
+     * @param b second data point
+     * @return area
+     */
+    public static double computeArea(Point a, Point b) {
+        final double x1 = a.getTimestamp();
+        final double x2 = b.getTimestamp();
+
+        if (x1 > x2) {
+            throw new IllegalArgumentException("timestamp of b needs to be greater than a");
+        }
+
+        if (x1 == x2) {
```

Comment:
One might want to perform a `long` comparison due to potentially problematic floating point comparisons.